### ISSUE TITLE
Ochronotic arthritis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10161]
**Preferred term label:**
Ochronosis HP:0030764

**Synonyms**
ochronotic osteoarthropathy 

**Definition (free text, please give PubMed ID)**
PMID: 19381613 Alkaptonuria is a rare disease in which the body does not have enough of an enzyme called homogentisic acid oxidase. Osteoarthritis is the most common degenerative joint disease. **Ochronotic arthritis** which resulting from the deposition of oxidized homogentisic acid within the connective tissues of peripheral joints has clinical feature that resembles those of osteoarthritis, but it has a unique manifestation

PMID: 32809406 Alkaptonuria has multiple clinical spectrums. Ochronosis, homogenistic aciduria, and **ochronotic osteoarthropathy** are called the triad of alkaptonuria.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Alkaptonuria ORPHA:56

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Not sure I understand the suggestion -- is there a change to the current definition?

[Ochronosis HP:0030764](https://hpo.jax.org/app/browse/term/HP:0030764)

**Comment by pnrobinson** [comment]
@carolina-f  Sorry I think I forgot to tag you in the above question.

**Comment by carolina-f** [comment]
I wanted to differentiate joint involvement from the accumulation of homogentisic acid in the connective tissues, but I believe that the term Ochronosis encompasses the concept.

**Comment by pnrobinson** [comment]
Thanks - I added "Ochronotic arthritis" as a synonym
